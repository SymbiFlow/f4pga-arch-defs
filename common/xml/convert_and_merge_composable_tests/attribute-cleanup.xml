<?xml version="1.0"?>
<!-- Clean up attributes
   * Normalize space around attributes on a tag
   * Sort the attributes by name
   * Strip xml:base attribute
  -->
<xml>
 <tag b="1"  c="123"  x=' abc' />
 <tag c="1"  d=" 123 "  x="abc" />
 <tag>
  <innertag   a2=" 2 "   b1="abc   "    a1="2">Hello</innertag>
 </tag>
</xml>
