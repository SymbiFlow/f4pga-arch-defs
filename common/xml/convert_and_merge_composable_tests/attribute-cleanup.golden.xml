<?xml version="1.0"?>
<xml>
  <tag b="1" c="123" x="abc"/>
  <tag c="1" d="123" x="abc"/>
  <tag>
    <innertag a1="2" a2="2" b1="abc">Hello</innertag>
  </tag>
</xml>
